### ISSUE TITLE
[orchestra2md] Incorrect conversion of appinfo element for FIXML

### ISSUE THREAD

**Issue by kleihan** [FIXTradingCommunity/tablature#68]
Tablature v1.0.2 does not process `<fixr:appinfo purpose="FIXML">` correctly. It creates a column "Fixml" with content `/P/[fixml:FIXMLencodingType: null]/P/` in markdown. Expected behavior is that it ignores such appinfo elements. They are not needed in markdown and are only for internal purposes to create the FIXML schemas.

**Comment by donmendelson** [comment]
No, appinfo is not *just* for internal use. It was intended that firms could extend their ROE with attributes of their own that are not predefined in Orchestra.

However, it would be possible to add a feature to suppress appinfo if desired.

**Comment by kleihan** [comment]
I was not precise enough when i said "such appinfo elements". I meant to say "appinfo elements with purpose FIXML". It is not about suppressing all appinfo elements. Tablature still has a problem with the conversion of appinfo that contains XML snippets it seems.

XML: `<fixml:FIXMLencodingType notReqXML="1" />`
MD: `/P/[fixml:FIXMLencodingType: null]/P/`

I do not expect to see "null" or square brackets. Is that intended?

**Comment by donmendelson** [comment]
An explanation for current ouput:
* The fixml element contents actually consists of 3 XML nodes.
* Invoking toString() results in a call to a Java library function to output a collection as a string. For each element, it outputs the node contents with type in brackets and any attributes, which in this case are null.
